src/services/rate-limit/index.ts

```diff
@@ -49,7 +49,7 @@ export const consumeLimiter = async ({
   keyToConsume,
 }: {
   rateLimitConfig: RateLimitConfig
-  keyToConsume: IpAddress | PhoneNumber | EmailAddress | AccountId
+  keyToConsume: IpAddress | PhoneNumber | EmailAddress | AccountId | ""
```

Comment:
keyToConsume must have a valid param, having `""` is not the intended behaviour

Reply:
cc @nicolasburtey